test/resources/SubscriptionSchedule.spec.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+var stripe = require('../../testUtils').getSpyableStripe();
+var expect = require('chai').expect;
+
+var SCHEDULE_TEST_ID = 'sub_sched_123';
+var REVISION_TEST_ID = 'sub_sched_rev_123';
+
+describe('Subscription Schedule Resource', function() {
+  describe('release', function() {
+    it('release a subscription schedule successfully', function() {
+      var data = {
+        invoice_now: true,
+      }
+      stripe.subscriptionSchedules.cancel(SCHEDULE_TEST_ID, data);
+      expect(stripe.LAST_REQUEST).to.deep.equal({
+        method: 'POST',
+        url: '/v1/subscription_schedules/' + SCHEDULE_TEST_ID + '/cancel',
+        data: data,
+        headers: {},
+      });
+    });
+  });
+
+  describe('create', function() {
+    it('Sends the correct request', function() {
+      var data = {
+        customer: 'cus_123',
+      };
+      stripe.subscriptionSchedules.create(data);
+      expect(stripe.LAST_REQUEST).to.deep.equal({
+        method: 'POST',
+        url: '/v1/subscription_schedules',
+        data: data,
+        headers: {},
+      });
+    });
+  });
+
+  describe('release', function() {
```

Comment:
You're testing `release` twice, and missing tests for `list`, `update` and `cancel`.